lib/helper.rb

```diff
@@ -0,0 +1,55 @@
+module Helper
+  def self.shape
+    shape = gets.chomp
+    while shape.size > 5 || shape.empty?
+      print
+      shape = gets.chomp
+    end
+    shape
+  end
+
+  def self.render_board(board, size)
+    board.each_with_index do |field, index|
+      print '|' unless (index % size).zero?
+      if field.nil?
+        print((index + 1).to_s.center(12))
+      else
+        print field.center(12)
+      end
+
+      next unless ((index + 1) % size).zero?
+
+      print "\n"
+      print '-' * (12 * size + (size - 1)) unless index == (size * size - 1)
+      print "\n"
+    end
+  end
```

Comment:
This would be simpler if you don't let the user decide the size of the board.
Return a string instead of using print on the `lib` folder.